.github/workflows/main.yml

```diff
@@ -312,15 +312,6 @@ jobs:
         run: |
           brew install automake autoconf-archive cmake libtool ninja
 
-      # libcxx build is not working with python >= 3.12
-      - name: Do not use preinstalled python >= 3.12
-        run: |
-          brew uninstall --ignore-dependencies python@3.12
-          [ -e /Library/Frameworks/Python.framework ] && \
-            sudo mv /Library/Frameworks/Python.framework /Library/Frameworks/Python.framework.dont-look-at-this
-          brew install python@3.11
-          ln -s "$(brew --prefix python@3.11)/bin/python3" "$(brew --prefix python@3.11)/bin/python"
-
       - name: Cache - vcpkg
         uses: actions/cache@v4
         with:
```

Comment:
Should also change cache key below, to something like `key: ${{ runner.os }}-vcpkg-1`